### ISSUE TITLE
Component router

### ISSUE THREAD

**Issue by nkbt** [edge/cronux#1]
See https://www.npmjs.com/package/component-router

Feels like this is exactly what is described in the README.

**Comment by alkhe** [comment]
Not really. I don't want to have a Router as a Component at all. It makes more sense to use an if or switch statement to determine what goes where.

Since I really think this has less to do with React and more to do with Flux.

**Comment by nkbt** [comment]
Components in React are just a nice way to do composition. Routing itself is just a flux stuff. You can use just a value and switch any way you want - see https://github.com/in-flux/component-router/issues/52

We also need some links - so some components will be there anyway.
You may not use links and simply listen to the Store and use ActionCreator for navigation. I mean it is quite flexible. Actually, most of the code is quite abstracted in a way you do not have to use router as a component. And could be abstracted further. 

Any input is really appreciated, thanks =)
